### PR TITLE
Bob rewrite -- draft

### PR DESCRIPTION
**Pull request by kytrinyx**
I didn't do a whole lot here, but I decided to ditch the word _lackadaisical_ as a lot of people who don't have English as a first language probably aren't familiar with it.

### CODE REVIEW

File: exercises/bob/introduction.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Introduction
+
+Bob is a teenager.
+He's not very smart, but he likes to think that he's very cool.
+He definitely doesn't get excited about things.
+That wouldn't be cool.
+
+When people talk to him, his responses are pretty limited.
+
+If you ask him a question, such as "How are you?", Bob answers 'Sure.'
```
Comment (ErikSchierboom) [inline exercises/bob/introduction.md:10]: Was the single/double quote inconsistency intended?

```suggestion
If you ask him a question, such as "How are you?", Bob answers "Sure."
```

File: exercises/bob/introduction.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Introduction
+
+Bob is a teenager.
+He's not very smart, but he likes to think that he's very cool.
```
Comment (ErikSchierboom) [inline exercises/bob/introduction.md:4]: I wonder if the "He's not very smart" part adds anything. It feels somewhat judgemental?

File: exercises/bob/introduction.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+If you ask him a question, such as "How are you?", Bob answers 'Sure.'
+
+He answers 'Whoa, chill out!' if you YELL AT HIM (in all capitals).
+
+He answers 'Calm down, I know what I'm doing!' if you yell a question at him.
+
+He says 'Fine. Be that way!' if you address him without actually saying anything.
+
+He answers 'Whatever.' to anything else.
```
Comment (ErikSchierboom) [inline exercises/bob/introduction.md:18]: Not saying this would be better, but did you consider making the five different types of responses into a list?

**Review by ErikSchierboom — APPROVED** [review]
Slightly said to see lackadaisical being removed, but I think it is the right choice at it is definitely something non-native speakers will struggle with.

File: exercises/bob/introduction.md (hunk 1 above)
Comment (kytrinyx) [reply exercises/bob/introduction.md:10]: Not intended! Nice catch, thank you.

File: exercises/bob/introduction.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+- _"Sure."_
+  This is his response if you ask him a question, such as "How are you?"
+- _"Whoa, chill out!"_
+  This is his answer if you YELL AT HIM (in all capitals).
+- _"Calm down, I know what I'm doing!"_
+  This is what he says if you yell a question at him.
+- _"Fine. Be that way!"_
+  This is how he responds if you address him without actually saying anything.
+- _"Whatever."_
+  This is what he answers to anything else.
```
Comment (ErikSchierboom) [inline exercises/bob/introduction.md:20]: Personally, I like this version better! It makes it more clear that there are exactly five response and their definitions are now grouped together, which I like.

File: exercises/bob/introduction.md (hunk 2 above)
Comment (iHiD) [reply exercises/bob/introduction.md:4]: Agreed.

```suggestion
He likes to think that he's very cool.
```

File: exercises/bob/introduction.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# Introduction
+
+Bob is a teenager.
+He's not very smart, but he likes to think that he's very cool.
+He definitely doesn't get excited about things.
```
Comment (iHiD) [inline exercises/bob/introduction.md:5]:
```suggestion
And he definitely doesn't get excited about things.
```

File: exercises/bob/introduction.md (hunk 4 above)
Comment (iHiD) [reply exercises/bob/introduction.md:20]: I have a preference for bold (`**`) not italic (`_`) formatting as I feel it makes it stand out more on the website. I think italics things tend to blur into the background more, so despite `_` being for emphasis, I actually think it has the opposite UX effect when rendered. So I'd change these to bold.

**Comment by kytrinyx** [comment]
Would you give this one another look?

**Comment by iHiD** [comment]
This is great.

I'm really torn on "lackadaisical". Because I consider it relatively iconic in Exercism terms (we even have a lackadaisical badge). I tried to find a video that would mean we could keep it and have a fun note about it and failed.

**Comment by kytrinyx** [comment]
> we even have a lackadaisical badge

I completely missed that. Oof. Yeah, if we could find a fun thing that explains lackadaisical that would work, but... hm. Could be tricky.

File: exercises/bob/introduction.md [hunk 6]
```diff
@@ -0,0 +1,8 @@
+# Introduction
+
+Bob is a teenager.
```
Comment (iHiD) [inline exercises/bob/introduction.md:3]:
```suggestion
Bob is a [lackadaisical](https://www.collinsdictionary.com/dictionary/english/lackadaisical) teenager.
```

This needs to use the non-inline markdown syntax though.

**Pull request closed without merging**